Controllers/postComments.controller.js

```diff
@@ -0,0 +1,12 @@
+const { sendComments } = require("../Models/sendComments.model");
+
+exports.postComments = (req, res, next) => {
+  const body = req.body.body;
+  const author = req.body.username;
+  const reviewID = req.params.review_id;
```

Comment:
I see what you mean, is it possible to get params through using body? I think I did it this way because earlier in the week I was told to pass through the values from the controller rather than reworking them in the model